week1/1-2/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>정우의 가계부</title>
+</head>
+<body>
+    <header>
+        <h1>정우의 가계부</h1>
+    </header>
+    <div class="my_asset_container">
+        <div class="asset_box">
+            <h2>나의 자산</h2>
+            <h1>135400</h1>
+            <div class="asset_inout">
+                <div class="asset_count">
+                    <img src="assets/plus_icon.png" alt="수입 아이콘">
+                    <p class="asset_in">400000</p>
+                </div>
+                <div class="asset_count">
+                    <img src="assets/minus_icon.png" alt="지출 아이콘">
+                    <p class="asset_out">264600</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="asset_history_container">
+        <div class="date_selector">
+            <img src="assets/arrow_left.png" alt="">
+            <p class="date">10월 11일</p>
+            <img src="assets/arrow_right.png" alt="">
+        </div>
+        <div class="history_list_container">
+            <div class="history_list_header">
+                <span>내역 리스트</span>
+                <div class="history_filter">
+                    <div>
+                        <input type="checkbox" name="in" checked>
+                        <label for="in">수입</label>
+                    </div>
+                    <div>
+                        <input type="checkbox" name="out" checked>
+                        <label for="out">지출</label>
+                    </div>
+                </div>
+            </div>
+            <div class="history_list">
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">식비</p>
+                    <h3 class="history_title">공차경기대후문점</h3>
+                    <p class="history_asset minus">-9000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">생필품</p>
+                    <h3 class="history_title">세븐일레븐</h3>
+                    <p class="history_asset minus">-1100</p>
+                </div>
+                <div class="history_item">
```

Comment:
```suggestion
            <section class="history_list">
             <ul>
                <li class="history_item">
                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
                    <p class="history_category">식비</p>
                    <h3 class="history_title">공차경기대후문점</h3>
                    <p class="history_asset minus">-9000</p>
                </li>
                <li class="history_item">
                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
                    <p class="history_category">생필품</p>
                    <h3 class="history_title">세븐일레븐</h3>
                    <p class="history_asset minus">-1100</p>
                </li>
```
이런식으로 시맨틱하게 만들어줄 수 있겠네요!